client/my-sites/stats/stats-views/months.jsx

```diff
@@ -138,7 +139,11 @@ const StatsViewsMonths = ( props ) => {
 
 			return (
 				<Month
-					href={ `/stats/month/${ siteSlug }?startDate=${ year }-${ month + 1 }-1` }
+					href={ `/stats/day/${ siteSlug }?chartStart=${ year }-${ month + 1 }-1&chartEnd=${ moment(
```

Comment:
Could we also use the format `YYYY-MM-DD` for the `chartStart` [and here](https://github.com/Automattic/wp-calypso/blob/9a0c231916418eb07f429edea0c8de980ba9827a/client/my-sites/stats/stats-views/months.jsx#L75) for consistency?

Reply:
Done, updated the date format everywhere to be consistent!